tests/gpu-tests/test_judge.py

```diff
@@ -66,3 +66,12 @@ def test_trtllm_judge():
     for line in lines:
         data = json.loads(line)
         assert 'judgement' in data
+
+    # Adding a summarization step to check that the results are formatted correctly
```

Comment:
did this actually fail without your fix? Just wondering if it's sufficient enough to catch an error or we need some other test cases for this